### PR TITLE
Update kubeedge roadmap

### PR DESCRIPTION
**Pull request by m1093782566**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

Update kubeedge roadmap.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
```
NONE
```

### CODE REVIEW

**Comment by m1093782566** [comment]
cc @CindyXing @qizha @kevin-wangzefeng @rohitsardesai83 @sids-b @skdwriting

File: README.md [hunk 1]
```diff
@@ -47,23 +47,41 @@ KubeEdge is composed of the following components:
...
+
 - Istio-based service mesh across Edge and Cloud where micro-services can communicate freely in the mesh.
 - Enhance performance and reliability of KubeEdge infrastructure.
 - Enable function as a service at the Edge.
 - Support more types of device protocols to Edge nodes such as AMQP, BlueTooth, ZigBee, etc.
 - Evaluate and enable much larger scale Edge clusters with thousands of Edge nodes and millions of devices.
 - Enable intelligent scheduling of applications to large scale Edge clusters.
+- Data management/analytics framework.
+
+**Release 0.3** - Plan to Release on 3 May 2019:
```
Comment (sids-b) [inline README.md]: 31st May, 3rdMay we plan for a beta release.

**Comment by m1093782566** [comment]
> I think in RoadMap we should have current and future planned activities. So can we remove about 0.1 and 0.2 release and add what we plan tentatively for 0.4 ??

I just refer to poseidon project roadmap in order to give a more clear picture of this project to new contributor: https://github.com/kubernetes-sigs/poseidon#roadmap. I can remove the v0.1 and v0.2 release if you think they are unnecessary.

File: README.md (hunk 1 above)
Comment (m1093782566) [reply README.md]: Done

**Comment by m1093782566** [comment]
Thanks @sids-b

**Pull request merged**